app/views/oauth_clients/index.html.erb

```diff
@@ -34,10 +34,14 @@
 <p><%= t(".no_apps_html", :oauth => link_to(t(".oauth"), "https://oauth.net")) %></p>
 <% else %>
 <p><%= t ".registered_apps" %></p>
-<% @client_applications.each do |client| %>
-  <div class="client_application">
-    <%= link_to client.name, :action => :show, :id => client.id %>
-  </div>
-<% end %>
+<p>
```

Comment:
Should be a list (which has automatic bottom padding) rather than wrapping in a paragraph

Reply:
Yeah. It probably isn't a list because people registering more than one application is very rare, but ok. Let me give that a try.